### DISCUSSION TITLE
Which behavior of the transform tool is a bug?

### DISCUSSION THREAD

**Discussion by Be-Cool-Man** [FreeCAD/FreeCAD#16016] [Discussions that will turn in to issues but need refining]
Hello community,

which of the following behaviors is a bug:
- Setting the Translation Increment to 0 freezes the object in place
- Loading a file, where the Translation Increment is set to 0, the Translation Increment is ignored and I can move the object freely
But, when I Load a file, where Translation Increment is greater than 0, the Translation Increment gets obeyed

Sincerely
Be-Cool-Man

**Comment by luzpaz** [comment]
@Be-Cool-Man please ask on the forum.freecad.org
